### PR TITLE
Update SignalRWorker

### PR DESCRIPTION
**Pull request by BrennanConroy**
@anurse @sebastienros

### CODE REVIEW

**Comment by BrennanConroy** [comment]
🆙 📅 
Just take a quick glance to make sure I didn't do anything weird

File: src/BenchmarksClient/Workers/SignalRWorker.cs [hunk 1]
```diff
@@ -159,23 +159,22 @@ public SignalRWorker(ClientJob job)
...
-
-                _stopped = true;
-
                 // stop connections
                 Log("Stopping connections");
                 var tasks = new List<Task>(_connections.Count);
                 foreach (var connection in _connections)
                 {
-                    tasks.Add(connection.StopAsync());
+                    tasks.Add(connection.DisposeAsync());
```
Comment (davidfowl) [inline src/BenchmarksClient/Workers/SignalRWorker.cs]: Because we never restart?

Comment (BrennanConroy) [reply src/BenchmarksClient/Workers/SignalRWorker.cs]: Yeah, each run is completely separate

Comment (davidfowl) [reply src/BenchmarksClient/Workers/SignalRWorker.cs]: Good this will free up http clients 😄

**Pull request merged**